modules/edit-modes/src/types.ts

```diff
@@ -106,7 +106,7 @@ export type GuideFeature = EditHandleFeature | TentativeFeature;
 export type GuideFeatureCollection = {
   type: 'FeatureCollection';
   features: Readonly<GuideFeature>[];
-  properties?: {};
+  properties?: object;
```

Comment:
object tends to be problematic as it doesn't recognize any fields. Perhaps...
```suggestion
  properties?: Record<string, unknown>;
```